syncmaster/server/settings/server/__init__.py

```diff
@@ -44,21 +44,21 @@ class ServerSettings(BaseModel):
     )
     session: SessionSettings = Field(
         default_factory=SessionSettings,  # type: ignore[arg-type]
-        description=":ref:`Session settings <backend-configuration-server-session>`",
+        description=":ref:`Session settings <server-configuration-server-session>`",
```

Comment:
Well, `backend-configuration-` prefix is probably okay for now, as we have different settings subsections for server, scheduler, worker and so on. If not, then we should replace `server-configuration-server-` with just `server-configuration-`

Reply:
Actually, now configuration links are inconsistent. There are `server-auth-providers`, `server-install`, `server-openapi`, but `backend-conguration-server-openapi`, `backend-configuration-server-session` and so on.

Probably we should use `server-configuration-openapi`, `server-configuration-session` instead.

Reply:
@IlyasDevelopment what do you think?

Reply:
Fixed. Now it looks more consistent